subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/service/TestRunnerWorkAction.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.instrumentation.service
+
+import com.avito.instrumentation.internal.InstrumentationTestsAction
+import org.gradle.api.provider.Property
+import org.gradle.workers.WorkAction
+import org.gradle.workers.WorkParameters
+
+@Suppress("UnstableApiUsage")
+public abstract class TestRunnerWorkAction : WorkAction<TestRunnerWorkAction.Params> {
```

Comment:
We could avoid serialization with `in memory work`

Reply:
yes, but why we need it? It should be serializable by gradle contract and i see no problem with it.
I understand that in case of inprocess strategy it is not necessary